gammapy/catalog/hawc.py

```diff
@@ -0,0 +1,134 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+HAWC 2FHL catalog
+
+TODO: Add reference_id to the paper?
```

Comment:
For the docstring at the top of `gammapy/catalog/hawc.py` I would suggest:
```
HAWC catalogs (https://www.hawc-observatory.org/)
```
Note that it doesn't appear in the HTML docs, so it's of limited use to put more info here.

You can check the new docs locally via
```
python setup.py buid_docs
make doc-show
```

I would suggest to put a link to ADS and paper reference for 2HWC in the class-level docstring of `SourceCatalog2HWC`.  If you don't mind doing it, it should really be added for the Fermi catalogs as well, and that would be a small and simple addition that you could just do in this PR now that we're noticing / discussing it.

Here's a good example of how to add a reference to a docstring:
* http://docs.gammapy.org/en/latest/api/gammapy.stats.significance_on_off.html
* http://docs.gammapy.org/en/latest/_modules/gammapy/stats/poisson.html#significance_on_off

Reply:
Fixed this.
Unfortunately, I can not build the documentation at my Laptop at home. A long error occurs.
On the office PC it worked yesterday. I don't know why.